src/assets/styles/components/_checkbox.css

```diff
@@ -1,8 +1,8 @@
 *:has(> [type="checkbox"]) {
-    display: flex;
     gap: calc(0.375rem * var(--fl-enhance-font-size-factor, 1));
-    align-items: flex-start;
+    list-style: none;
     margin-bottom: 0.75rem;
+    text-indent: -0.5rem;
```

Comment:
I would expect the second line to line up with the first as indicated here:
<img width="285" alt="Screenshot 2025-04-22 at 1 00 28 PM" src="https://github.com/user-attachments/assets/fa889a68-68e2-42c3-8ea5-3c625af5632c" />